lib/src/main/java/com/orange/ods/compose/component/textfield/OdsTextFieldsCommon.kt

```diff
@@ -171,19 +172,26 @@ internal class OdsExposedDropdownMenuTrailing(val expanded: Boolean) : OdsTextFi
     @Composable
     override fun Content(modifier: Modifier) {
         val degrees = if (expanded && extraParameters.enabled) 180f else 0f
-        Box(modifier = modifier.rotate(degrees)) {
-            OdsTextFieldIcon(
-                painter = rememberVectorPainter(image = Icons.Filled.ArrowDropDown),
-                contentDescription = null,
-                onClick = if (extraParameters.enabled) {
-                    {}
-                } else {
-                    null
-                }
-            )
+        val onClick = if (extraParameters.enabled) {
+            {}
+        } else {
+            null
         }
-    }
 
+        Box(
+            modifier = Modifier
+                .rotate(degrees)
+        ) {
+            OdsIconButton(
+                modifier = Modifier.focusProperties { canFocus = false },
+                icon = OdsIconButton.Icon(
+                    painter = rememberVectorPainter(image = Icons.Filled.ArrowDropDown),
+                    contentDescription = ""
+                ),
+                enabled = onClick != null,
+                onClick = onClick ?: {})
```

Comment:
If I'm correct `onClick ?: {}` is always equal to `{}` and thus this can be simplified:

```kotlin
OdsIconButton(
    modifier = Modifier.focusProperties { canFocus = false },
    icon = OdsIconButton.Icon(
        painter = rememberVectorPainter(image = Icons.Filled.ArrowDropDown),
        contentDescription = ""
    ),
    enabled = extraParameters.enabled,
    onClick = {})
```